index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700;800;900&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <div class="header-image navbar-item"><img src="images/image_geometry_1.svg" alt="A geometry image"/></div>
+      <div class="navbar-item icon"><i class="fas fa-bars "></i></div>
+      <ul>
+        <li><a href="#">Hello</a></li>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
```

Comment:
- [ ] I can't see this anywhere, please fix it, and if you are doing the mobile v you can remove them .